src/OrchardCore/OrchardCore.DisplayManagement/Shapes/Shape.cs

```diff
@@ -65,13 +65,13 @@ public virtual ValueTask<IShape> AddAsync(object item, string position)
 
             _items ??= [];
 
-            if (item is IHtmlContent)
+            if (item is IHtmlContent htmlContent)
```

Comment:
Can't this whole if/else/else be a single TryWrap?

Reply:
updated